modulesets/bootstrap.modules

```diff
@@ -21,19 +21,25 @@
   <repository type="tarball" name="intltool"
               href="http://launchpad.net/intltool/trunk/"/>
 
-  <autotools id="make">
+  <autotools id="make" bootstrap="true">
```

Comment:
1. that's how updated `bootstrap.modules` looks like: https://git.gnome.org/browse/gtk-osx/tree/modulesets-stable/bootstrap.modules#n24; the whole file has been updated
2. here's the explanation: https://git.gnome.org/browse/jhbuild/tree/jhbuild/modtypes/autotools.py#n372